prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1092,6 +1187,14 @@ static int rxr_ep_setopt(fid_t fid, int level, int optname,
 		if (ret)
 			return ret;
 		break;
+	case FI_OPT_CUDA_API_PERMITTED:
+		if (optlen != sizeof(bool))
+			return -FI_EINVAL;
+		boolval = *(bool *)optval;
+		ret = rxr_ep_set_cuda_api_permitted(rxr_ep, boolval);
```

Comment:
IMO having boolval is not helpful. Maybe simply `ret = rxr_ep_set_cuda_api_permitted(rxr_ep, *(bool *)optval);`

Reply:
fixied in new revision.